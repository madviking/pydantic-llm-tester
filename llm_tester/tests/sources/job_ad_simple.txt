Software Developer - Python

TechCorp Inc. is looking for a skilled Software Developer to join our growing team in San Francisco, CA. 

About the Role:
We are seeking an experienced Python developer to help build and maintain our cloud-based applications. This is a full-time position with a competitive salary range of $110,000 - $140,000 per year depending on experience.

Requirements:
- 3+ years of professional Python development experience
- Strong knowledge of SQL and database design
- Experience with Flask or Django web frameworks
- Familiarity with cloud platforms (AWS, GCP, or Azure)
- Understanding of RESTful API design
- Good communication skills

The ideal candidate will be self-motivated, detail-oriented, and able to work effectively in a team environment. This is not a remote position, and candidates must be able to work from our San Francisco office.

Apply now to join our innovative team and help shape the future of our technology platforms!