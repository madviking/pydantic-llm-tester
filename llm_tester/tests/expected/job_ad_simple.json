{
  "title": "Software Developer - Python",
  "company": "TechCorp Inc.",
  "location": "San Francisco, CA",
  "salary_range": "$110,000 - $140,000 per year",
  "experience_required": "3+ years of professional Python development experience",
  "skills": [
    "Python",
    "SQL",
    "database design",
    "Flask or Django",
    "cloud platforms (AWS, GCP, or Azure)",
    "RESTful API design",
    "communication skills"
  ],
  "description": "We are seeking an experienced Python developer to help build and maintain our cloud-based applications.",
  "remote": false
}